int64 pixel_x
int64 pixel_y
---
float32 pose_x
float32 pose_y
float32 pose_z